guide/popular-topics/errors.md

```diff
@@ -144,10 +144,11 @@ const { Client, Intents } = require('discord.js');
 // Should not be here!
 const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES] });
 
-module.exports = (message, args) => {
-	// Should be message.client instead!
-	client.users.fetch(args[0]).then(user => {
-		message.reply(`Your requested user: ${user.tag}`);
+module.exports = (interaction) => {
+	const { name: id } = interaction.options.get('target');
```

Comment:
`options.getString(...)` is a method, aswell as the required parameter (assuming this would be a required parameter)